data.sql

```diff
@@ -4,3 +4,11 @@ INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALU
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Gabumon','2018-11-15',2,true,8.0);
```

Comment:
Great job @Htetaungkyaw71 🤝

- [ ] Kindly take note of the negative values for the `weight_kg` column. make sure all the values are entered as they are given in the project requirements.